code/modules/projectiles/projectile.dm

```diff
@@ -1061,6 +1061,7 @@
 	if(!shrapnel_type || !LAZYLEN(embedding))
 		return
 
+	// sigh
```

Comment:
Why touch projectile.dm just to add a comment that says sigh

Reply:
bro did you see the line below it

Reply:
It's still cringe as fuck to do nothing, explain nothing, and touch a file to explicitly state that nothing.

Reply:
removed your highness